tests/utils/test_cache_utils.py

```diff
@@ -303,6 +306,29 @@ def test_cache_extra_left_padding(self, cache_implementation):
         decoded = self.tokenizer.batch_decode(gen_out, skip_special_tokens=True)
         self.assertListEqual(decoded, EXPECTED_GENERATION)
 
+    @parameterized.expand(TEST_CACHE_IMPLEMENTATIONS)
+    def test_cache_pipe_rope_model(self, cache_implementation):
+        """Tests caches with a RoPE model"""
```

Comment:
Let's be more detailed in test docstrings. This statement is also true for the other tests in this class (they also use a model with RoPE), so the test is for a problem more specific than that :p

It also helps tagging the original issue, if it is a regression test (e.g. "Regression test for #<issue/pr number>")